### PR TITLE
Implement basic login

### PR DESCRIPTION
**Pull request by namkidd**
ready

### CODE REVIEW

**Comment by namkidd** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in(user)
```
Comment (hungnh103) [inline app/helpers/sessions_helper.rb]: remove `()`

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (hungnh103) [inline app/helpers/sessions_helper.rb]: `current_user.present?`

**Comment by hungnh103** [comment]
commented
Em fix xong rồi merge nhé

**Pull request merged**